fees/suilend/index.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  Adapter,
+  FetchResultFees,
+} from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import fetchURL from "../../utils/fetchURL";
+
+const suilendFeesURL = 'https://api.suilend.fi/stats/daily-fees';
+
+interface DailyStats {
+  date: string;
+  start: number;
+  end: number;
+  borrowFees: number;
+  borrowInterestPaid: number;
+  protocolFees: number;
+  liquidatorBonuses: number;
+  liquidationProtocolFees: number;
+  previous: string;
+  next: string;
+}
+
+const methodology = {
+  Fees: 'Interest and fees paid by borrowers and the liquidated',
+  ProtocolReveneue: 'The portion of the total fees going to the Suilend treasury'
+}
+
+const fetchSuilendStats = async (timestamp: number): Promise<FetchResultFees> => {
+  const url = `${suilendFeesURL}?ts=${timestamp}`
```

Comment:
this timestamp is end of 24 hour period or start of?

Reply:
For whatever timestamp is passed in, it returns fees that fall within the 24 UTC date boundaries that the passed in timestamp is contained in